### PR TITLE
Add integration tests for contracts originated with wallet api

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Adding more integration tests that mimic contract integration tests but using the wallet API.

Before submitting this PR, please make sure:

- [x] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [x] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes.

### CODE REVIEW

File: integration-tests/wallet-estimation-tests.spec.ts [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+    const amt = 2000000 + DEFAULT_FEE.REVEAL;
+
+    beforeAll(async (done) => {
+      try {
+        await setup()
+        LowAmountTez = await createAddress();
+        const pkh = await LowAmountTez.signer.publicKeyHash()
+        const transfer = await Tezos.wallet.transfer({ to: pkh, mutez: true, amount: amt }).send();
+        await transfer.confirmation();
+        const op = await Tezos.contract.originate({
```
Comment (roxaneletourneau) [inline integration-tests/wallet-estimation-tests.spec.ts:27]: Can we use the wallet API instead?

File: integration-tests/wallet-estimation-tests.spec.ts [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+          balance: "1",
+          code: managerCode,
+          init: { "string": pkh },
+        })
+        contract = await op.contract()
+        contract = await LowAmountTez.contract.at(contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/wallet-estimation-tests.spec.ts:33]: Can we use the wallet API instead?

File: integration-tests/wallet-estimation-tests.spec.ts [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+
+  describe(`Estimate with very low balance using: ${rpc}`, () => {
+    let LowAmountTez: TezosToolkit;
+    const amt = 2000 + DEFAULT_FEE.REVEAL;
+
+    beforeAll(async (done) => {
+      await setup()
+      LowAmountTez = await createAddress();
+      const pkh = await LowAmountTez.signer.publicKeyHash()
+      const transfer = await Tezos.contract.transfer({ to: pkh, mutez: true, amount: amt });
```
Comment (roxaneletourneau) [inline integration-tests/wallet-estimation-tests.spec.ts:418]: Can we use the wallet API instead?

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: integration-tests/wallet-estimation-tests.spec.ts (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/wallet-estimation-tests.spec.ts:27]: whoops, I see, thanks 👍🏻 )

Comment (michaelkernaghan) [reply integration-tests/wallet-estimation-tests.spec.ts:27]: changed to use the wallet api throughout

File: integration-tests/wallet-estimation-tests.spec.ts (hunk 3 above)
Comment (michaelkernaghan) [reply integration-tests/wallet-estimation-tests.spec.ts:418]: changed to use Wallet API.

File: integration-tests/wallet-estimation-tests.spec.ts (hunk 2 above)
Comment (michaelkernaghan) [reply integration-tests/wallet-estimation-tests.spec.ts:33]: Changed to use wallet API.

**Review by roxaneletourneau — APPROVED** [review]

**Pull request merged**